hw/ip/prim/prim_ram_1p_scr.core

```diff
@@ -15,7 +15,6 @@ filesets:
       - lowrisc:prim:buf
       - lowrisc:prim:cipher
       - lowrisc:prim:util_get_scramble_params
-      - lowrisc:dv_verilator:memutil_dpi_scrambled
```

Comment:
This one looks good to me. The corresponding code is only used inside `hw/ip/otbn/dv/memutil/otbn_memutil.h` which has an explicit dependency on this core.